my_project/global_scheme.py

```diff
@@ -895,6 +895,57 @@
     "utci_noSun_Wind",
     "utci_noSun_noWind",
 ]
+variables_sun_path_chart_dropdown = [
```

Comment:
Instead of creating a new dictionary from scratch could you please make a deep copy of the dictionaries above (variables_dropdown and variables_more_variables_dropdown) and remove the variables you do not need so we avoid code duplications?